src/createBlock.ts

```diff
@@ -118,6 +121,15 @@ function formatData(data) {
     return JSON.stringify(data, null, 4);
 }
 
+/**
+ * Write the block to a file.
+ */
+function writeBlockDataToLocalFile(blockData, multiHash, formattedData) {
+    const filename = `block-${blockData.sequence}-${multiHash}.json`;
+    fs.writeFileSync(filename, formattedData);
```

Comment:
Best practice probably to use fs.writeFile and make this function async, though that would require figuring out `util.promisify` which only works in certain versions of Node. Maybe just put a TODO?

Reply:
If you want it to be cross platform you can use `bluebird.promisify` and add `bluebird` as a dep

Reply:
I manually created and returned a promise. Is it better to use promisify?